### ISSUE TITLE
Prepare for new domain

### ISSUE THREAD

**Issue by JRegimbal** [Shared-Reality-Lab/IMAGE-browser#34]
Once the server and website are ready, the extension should send all requests to the new domain (`image.a11y.mcgill.ca`) possibly with a new endpoint. Additionally, the website should be linked to as a homepage in the manifest.json.
